packages/core/src/layout/Sidebar/UserSettings.tsx

```diff
@@ -69,6 +70,11 @@ export function SidebarUserSettings() {
           apiRef={oktaAuthApiRef}
           icon={Star}
         />
+        <OIDCProviderSettings
```

Comment:
Just realized this triggers the same question as for the access token, as in: Is there any value for a plugin to be able to request an "auth0" ID token? Afaik the ID token you get depends entirely on what provider you configured in auth0, so there is no predictability to what you could do with it.

I think it would be possible to allow the use of auth0 to implement most, if not all of the other auth providers that we currently support. Not something I think we should cover in this PR though, and tbh its probably going to add to the confusion around auth as well :grin:.

Either way, it's probably best to remove the `OpenIdConnectApi` implementation from `Auth0Auth` as well, along with this entry in the sidebar. The way you'd log out after signing in with auth0 is the logout button at the bottom.